Python 3.8.10 (default, Nov 22 2023, 10:22:35)
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> add_integer = __import__('0-add_integer').add_integer
>>> add_integer(2, 3)
5
>>> add_integer(2, -3)
-1
>>> add_integer(-2, -3)
-5
>>> add_integer(-2, "and")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/abdulquyum/alx_projects/alx-higher_level_programming/0x07-python-test_driven_development/0-add_integer.py", line 10, in add_integer
    raise TypeError("b must be an integer")
TypeError: b must be an integer
>>> add_integer(True, 3)
4
>>> add_integer("you", 3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/abdulquyum/alx_projects/alx-higher_level_programming/0x07-python-test_driven_development/0-add_integer.py", line 8, in add_integer
    raise TypeError("a must be an integer")
TypeError: a must be an integer
>>> add_integer("you", 3.7)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/abdulquyum/alx_projects/alx-higher_level_programming/0x07-python-test_driven_development/0-add_integer.py", line 8, in add_integer
    raise TypeError("a must be an integer")
TypeError: a must be an integer
